### PR TITLE
Add oom unit tests for load and invoke

### PR DESCRIPTION
**Pull request by rohithkrn**
## Description ##

Add oom unit tests for load and invoke

### CODE REVIEW

File: engines/python/src/test/resources/echo/model.py [hunk 1]
```diff
@@ -39,6 +42,8 @@ def handle(inputs: Input):
         return "invalid_type"
     elif inputs.contains_key("exit"):
         sys.exit()
+    elif inputs.contains_key("OOM"):
+        raise OutOfMemoryError
```
Comment (frankfliu) [inline engines/python/src/test/resources/echo/model.py:46]: You can just use built-in exception MemoryError

File: engines/python/src/test/resources/oom/serving.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+option.pythonExecutable=python3
```
Comment (frankfliu) [inline engines/python/src/test/resources/oom/serving.properties:1]: this file is unnecessary, we can remove it

File: engines/python/src/test/resources/oom/model.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    pass
+
+
+def handle(inputs: Input):
+    """
+    Default handler function
+    """
+
+    if inputs.is_empty():
+        raise OutOfMemoryError()
```
Comment (frankfliu) [inline engines/python/src/test/resources/oom/model.py:35]:
```suggestion
        raise MemoryError()
```

File: engines/python/src/test/resources/echo/model.py (hunk 1 above)
Comment (rohithkrn) [reply engines/python/src/test/resources/echo/model.py:46]: pytorch throws `OutOfMemoryError`, so thought it makes more sense to test that particular case.

File: engines/python/src/test/resources/oom/serving.properties (hunk 2 above)
Comment (rohithkrn) [reply engines/python/src/test/resources/oom/serving.properties]: updated.

File: engines/python/src/test/resources/echo/model.py (hunk 1 above)
Comment (frankfliu) [reply engines/python/src/test/resources/echo/model.py:46]: This is mock test anyway. No difference between built-in exception vs customized class

Comment (rohithkrn) [reply engines/python/src/test/resources/echo/model.py]: Updated to keep it clean. Although there's a minor difference, when code change happens for some reason where we check for `OutOfMemoryError`, this test catches incorrect changes.

**Review by frankfliu — APPROVED** [review]

**Pull request merged**